plugins/Jobs.md

```diff
@@ -24,7 +24,14 @@ Bedrock::Jobs is a plugin to the [Bedrock data foundation](../README.md) that ma
    * *data* - (optional) New data object to associate with the job (especially useful if repeating, to pass state to the next worker).
 
  * **DeleteJob( jobID )** - Removes all trace of a job.
-   * *jobID* - Identifier of the job to delete 
+   * *jobID* - Identifier of the job to delete
+
+ * **Retry( jobID )** - Removes all trace of a job.
+   * *jobID* - Identifier of the job to retry
+   * *delay* - (optional) Number of seconds to wait before retrying
```

Comment:
I'd add "if the job has a `repeat` value set or `nextRun` is sent, then this param is ignored.